examples/md-flexible/src/domainDecomposition/RegularGridDecomposition.cpp

```diff
@@ -262,28 +271,38 @@ void RegularGridDecomposition::reflectParticlesAtBoundaries(AutoPasType &autoPas
     auto reflect = [&](bool isUpper) {
       for (auto p = autoPasContainer.getRegionIterator(reflSkinMin, reflSkinMax, autopas::IteratorBehavior::owned);
            p.isValid(); ++p) {
-        auto vel = p->getV();
-        // reverse velocity in dimension if towards boundary
-        if ((vel[dimensionIndex] < 0) xor isUpper) {
-          vel[dimensionIndex] *= -1;
+        // Check that particle is within 6th root of 2 * sigma
+        const auto position = p->getR();
+        const auto distanceToBoundary = isUpper ? reflSkinMax[dimensionIndex] - position[dimensionIndex]
+                                                : position[dimensionIndex] - reflSkinMin[dimensionIndex];
```

Comment:
```suggestion
        const auto distanceToBoundary = std::abs(reflSkinMax[dimensionIndex] - position[dimensionIndex]);
```

Reply:
Note, this is not just taking the absolute but takes distance to the two different boundaries, so std::abs will only work in conjunction with the conditional operator, at which point there is no gain in readability (and maybe even a loss).

I can add a comment explaining isUpper, or explaining this step if you feel it is needed.

Reply:
I overlooked `Min` vs `Max`...